args_parser.go

```diff
@@ -142,6 +144,10 @@ func newKingpinParser() argsParser {
 			return nil
 		}).
 		Bool()
+	app.Flag("scriptPath", "Lua script Path").
```

Comment:
This, probably, better be `script-path`, to be in accordance with naming convention used for flag names, i.e. `no-print` (dash(?)/kebab(??)-case, according to wiki) and not `noPrint` (CamelCase).